opentracing.podspec.src

```diff
@@ -1,19 +0,0 @@
-Pod::Spec.new do |s|
```

Comment:
this file is used to generate the `.podspec` (namely its `version`), so please make the edit here.